documentation/install_from_source.markdown

```diff
@@ -8,6 +8,7 @@ layout: default
 [configure_heap]: ./configure.html#configuring-the-java-heap-size
 [module]: ./install_via_module.html
 [packages]: ./install_from_packages.html
+[running_tests]: /CONTRIBUTING.html#running-the-tests
```

Comment:
Same question here...the `CONTRIBUTING.md` doc is located in the root dir of the project, where as the rest are within `documentation`. I could move that into the documentation folder if that's easier.